### DISCUSSION TITLE
Exposing code-server from remote host to the internet.

### DISCUSSION THREAD

**Discussion by arkadiusz-czerwinski** [coder/code-server#4812] [Q&A]
Hello, for now, I want to expose my code server to word. It works for docker, works for ngrok proxy, doesn't work with --host=0.0.0.0 argument in the command line. How to tackle this problem, is it intendent?

**Comment by jsjoeio** [comment]
Have you read [this guide](https://coder.com/docs/code-server/latest/guide)?
